README.md

```diff
@@ -55,10 +55,13 @@ Triton Inference Server.
   example, you can specify a latency budget to filter out model configurations
   that do not satisfy the specified latency threshold.
 
+## Examples and Tutorials
+
+See our [Quick Start](docs/quick_start.md) for a guide of how to use Model Analyzer to profile, analyze and report on a simple PyTorch model.
```

Comment:
One thing that would be nice is a very short quick start here on this front page that contains just the command lines needed to get the example to work and refer to the guide afterwords. See [Triton's Readme](https://github.com/triton-inference-server/server#serve-a-model-in-3-easy-steps) for an example.

This is the first thing users see when going to the repo, so it should be a quick and easy example.